### PR TITLE
Binaries to the CDN registry at tag release

### PR DESCRIPTION
**Pull request by luis02lopez**
Add step to additionally upload the binary to the CDN. We need to define what we consider as "latest" is the latests on pre-release or the one stable?

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -66,6 +71,11 @@ jobs:
           -H "Content-Type: $(file -b --mime-type build/lavad)" \
           --data-binary @build/lavad \
           $(echo '${{ github.event.release.upload_url }}' | sed 's/{?name,label}/?name=lavad-${{ github.event.release.tag_name }}-linux-amd64/g')
+
+          #Upload to binary registry
+          aws s3 cp build/lavad s3://network-upgrades-zarak/"${{ github.event.release.tag_name }}"/linux-amd64/lavad
+          #Latest -- We need to define what is the latest for us
+          # aws s3 cp build/lavad s3://network-upgrades-zarak/latest/linux-amd64/lavad
```
Comment (luis02lopez) [inline .github/workflows/release.yml:78]: We need to define what we consider as "latest" is the latests on pre-release or the one stable?

This automation is triggered on modification of any Release entry, if we consider latest being the higher tag number, I'll be adding that condition.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -7,6 +7,11 @@ on:
 permissions:
   contents: write
 
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```
Comment (luis02lopez) [inline .github/workflows/release.yml:11]: We'll need to add these keys to the project. After the PR has been approved, I'll ask to have admin priviledges on this repo to add the sensitive information into the secrets.

Comment (Yaroms) [reply .github/workflows/release.yml:11]: @luis02lopez @praveenperera , any follow up on that?

Comment (praveenperera) [reply .github/workflows/release.yml:11]: @luis02lopez you can use

```
          role-to-assume: arn:aws:iam::529724559987:role/github-cdk-ci-ecr-role-repo-lava
```

**Pull request closed without merging**